silverback/_cli.py

```diff
@@ -236,6 +241,42 @@ def workspaces(platform: PlatformClient):
         )
 
 
+@workspaces.command(name="new", section="Platform Commands (https://silverback.apeworx.io)")
+@click.option(
+    "-n",
+    "--name",
+    "workspace_name",
+    help="Name for new workspace",
+)
+@click.option(
+    "-s",
+    "--slug",
+    "workspace_slug",
+    help="Slug for new workspace",
+)
+@platform_client
+def new_workspace(
+    platform: PlatformClient,
+    workspace_name: str,
+    workspace_slug: str,
+):
+    """Create a new workspace"""
+
+    if workspace_name:
+        click.echo(f"name: {workspace_name}")
+        click.echo(f"slug: {workspace_slug or workspace_name.lower().replace(' ', '-')}")
+
+    elif workspace_slug:
+        click.echo(f"slug: {workspace_slug}")
+
+    workspace = platform.create_workspace(
+        workspace_name=workspace_name,
+        workspace_slug=workspace_slug,
+    )
+    if workspace.created:
```

Comment:
Not sure what this is, but would advise just to re-raise the error the platform gives you

Reply:
This shows the success message after creating the workspace. 
```
$ silverback cluster workspaces new -p staging -n testing2 -s testing2
name: testing2
slug: testing2
SUCCESS: Created 'testing2'
```

Reply:
Was the `.created` property already in there?

Reply:
The `.created` property is only there after `workspace = platform.create_workspace()` since it shows the `datetime` of when the workspace was created.

Reply:
Ahhh I see what you are doing, it will actually raise if it fails to deploy, you don't need `if workspace.created`

Reply:
Should I still have a success message to let the user know that it was deployed or is that not necessary? We have a success message with making new clusters so I thought we should keep it the same with workspaces.

Reply:
Yes, you just don't need the if statement, it will always be true if you make it that far